.github/workflows/main.yml

```diff
@@ -5,40 +5,23 @@ on:
       - master
   pull_request:
     branches:
-      - '**'
+      - master
```

Comment:
isn't ci suppose to run even if you open PR against some old branch. For example if backporting fix to older version?

Reply:
Yes, I'll revert this change.  Probably worth opening a PR to fix this in the automated CI flows too, [the line is here](https://github.com/protocol/.github/blob/master/templates/.github/workflows/js-test-and-release.yml#L8).